pycbc/inference/models/relbin.py

```diff
@@ -596,10 +596,11 @@ def _loglr(self):
             filt += filter_i
             norm += norm_i
 
+        loglr = self.marginalize_loglr(filt, norm)
         if self.return_sh_hh:
```

Comment:
Why do we need this if statement? Shouldn't the existing flag already used for demarginalization take care of this?  E.g. why not use the reconstruct_phase flag?

https://github.com/gwastro/pycbc/blob/master/pycbc/inference/models/tools.py#L241

Reply:
@ahnitz This is used here https://github.com/WuShichao/pycbc/blob/accelerate_multiband/pycbc/inference/models/hierarchical.py#L752 We need to get LISA's sh and hh.